src/PrestaShopBundle/Controller/Admin/SpecificPriceController.php

```diff
@@ -96,6 +101,78 @@ public function addAction(Request $request)
         return $response;
     }
 
+    /**
+     * Get one specific price list for a product
+     *
+     * @Template("@PrestaShop/Admin/Product/ProductPage/Forms/form_specific_price.html.twig")
+     *
+     * @param int $idSpecificPrice
+     *
+     * @return array
+     */
+    public function getUpdateFormAction($idSpecificPrice)
+    {
+        /** @var AdminProductWrapper $adminProductWrapper */
+        $adminProductWrapper = $this->get('prestashop.adapter.admin.wrapper.product');
+        try {
+            $price = $adminProductWrapper->getSpecificPriceDataById($idSpecificPrice);
+        } catch (\PrestaShopObjectNotFoundException $e) {
+            $message = $this->trans('Cannot find specific price %*price', 'Admin.Catalog.Notification', ['price' => $idSpecificPrice]);
```

Comment:
```php
$message = $this->trans(
    'Cannot find specific price %*price', 
    'Admin.Catalog.Notification', 
    ['price' => $idSpecificPrice]
);
```

Reply:
I believe @eternoendless told me once that the translations tool is unable to parse this multi-line syntax for `trans()` . Pablo, can you confirm ?

Reply:
Nope it can, it use PhpParser =)

Reply:
Yes it works on multiline, I have added a test for that.

It doesn't work for any case that isn't static, for example:

```php
$this->trans('text', [], $domain);
$this->someWrapperForTrans('text', [], 'domain');
call_user_function_array([$this, 'trans'], $parameters);
```

Reply:
Btw looks like your two last parameters are inverted, it usually is "string", "replacements", "domain".

Also, what's the deal with `%*price`? Shouldn't it be `%price%` instead?

Reply:
This is the signature of `FrameworkBundleAdminController::trans()`:
https://github.com/PrestaShop/PrestaShop/blob/develop/src/PrestaShopBundle/Controller/Admin/FrameworkBundleAdminController.php#L266

Is it considered a wrapper for trans ?

Reply:
It's a wrapper for trans, and it works in both situation, we changed it two months ago :)
https://github.com/PrestaShop/TranslationToolsBundle/commit/2430934ad376aff0c17bd8ff9ecb19ce9b14eb05

But agreed with @eternoendless  what is `%*price`?

Reply:
`%*price` is a mistake 😝